h2d/Text.hx

```diff
@@ -371,10 +382,11 @@ class Text extends Drawable {
 				}
 				else wLastSep = size;
 			}
-			else if( (x + esize + letterSpacing) - wLastSep > maxWidth ) {
+			else if( wordWrap && (x + esize + letterSpacing) - wLastSep > (maxWidth-esize) ) {

```

Comment:
Why the `-esize` change, though? I'm now getting letters thrown to next line if the character ends exactly at max width